ClientApp/src/Pages/GettingStarted/GettingStarted.tsx

```diff
@@ -0,0 +1,104 @@
+
+import React, { useState } from 'react';
+import LocationData from './LocationData.json';
+import './GettingStarted.scss';
+
+import { LocationSummary } from './LocationSummary';
+import ReactWeather, { useOpenWeather } from 'react-open-weather';//ignore red-line, it still works
+
+enum Locations {
+    Scotland,
+    Canada,
+    Australia
+}
+
+function GettingStarted() {
+    const [selectedLocation, setSelectedLocation] = useState(Locations.Scotland);
+
+    let x;
+    let y;
+    let loc;
+
+    const latScotland = '57.7207794189';
+    const LonScotland = '-5.6862931252';
+    const LatCanada = '53.135509';
+    const LonCanada = '-57.660435';
+    const LatAustralia = '-20.917574';
+    const LonAustralia = '142.702789';
+
+    if (selectedLocation === Locations.Canada) {
+        x = LatCanada;
+        y = LonCanada;
+        loc = " NewFoundland, Canada";
+    }
+    else if (selectedLocation === Locations.Scotland) {
+        x = latScotland;
+        y = LonScotland;
+        loc = "Gairloch, Scotland"
+    }
+    else if (selectedLocation === Locations.Australia) {
+        x = LatAustralia;
+        y = LonAustralia;
+        loc = "Queensland, Australia"
+    }
+
+    const { data, isLoading, errorMessage } = useOpenWeather({
+        key: '43887186b07bb5deaad592918ae44878',
```

Comment:
this looks like it might be a secret API key...
Are you sure its OK for it to be made public in GitHub?

(if not you might need to cancel this one and generate a new one)